src/main/java/com/revlearn/team1/initializer/DataInitializer.java

```diff
@@ -78,6 +79,13 @@ private void createInitialTransactions(JsonNode transactionsNode) {
         }
     }
 
+    private void createInitialModules(JsonNode modulesNode) {
```

Comment:
haha im surprised the dataInitializer didn't blow up on you! that shit is finicky af